spec/src/main/asciidoc/ch08-entity-packaging.adoc

```diff
@@ -4,127 +4,129 @@
-to specify managed persistence classes included in the persistence unit,
-object/relational mapping information for those classes, scripts for use
-in schema generation and the bulk loading of data, and other
-configuration information for the persistence unit and for the entity
-manager(s) and entity manager factory for the persistence unit. This
-information may be defined by containment or by reference, as described
+A _persistence.xml_ file defines a persistence unit. The _persistence.xml_
+file is located in the _META-INF_ directory of the root of the persistence
+unit. It may be used to specify:
+
+- managed persistence classes included in the persistence unit,
+- object/relational mapping information for those classes,
+- scripts for use in schema generation and bulk loading of data, and
+- other configuration information for the persistence unit and for the
+  entity managers and entity manager factory of the persistence unit.
+
+This information may be defined by containment or by reference, as described
 below.
 
-The object/relational mapping information can
-take the form of annotations on the managed persistence classes included
-in the persistence unit, an _orm.xml_ file contained in the _META-INF_
-directory of the root of the persistence unit, one or more XML files on
-the classpath and referenced from the _persistence.xml_ file, or a
-combination of these.
-
-The managed persistence classes may either be
-contained within the root of the persistence unit; or they may be
-specified by reference—i.e., by naming the classes, class archives, or
-XML mapping files (which in turn reference classes) that are accessible
-on the application classpath; or they may be specified by some
-combination of these means. See <<a12305>>.
-
-The root element of the _persistence.xml_
-file is the _persistence_ element. The _persistence_ element consists of
-one or more _persistence-unit_ elements.
-
-The _persistence-unit_ element consists of
-the _name_ and _transaction-type_ attributes and the following
-sub-elements: _description_, _provider_, _jta-data-source_,
-_non-jta-data-source_, _mapping-file_, _jar-file_, _class_,
-_exclude-unlisted-classes_, _shared-cache-mode_, _validation-mode_,
+The object/relational mapping information can take the form of:
+
+- annotations on the managed persistence classes included in the persistence
+  unit,
+- an _orm.xml_ file contained in the _META-INF_ directory of the root of the
+  persistence unit,
+- one or more XML files accessible on the classpath and referenced from the
+  _persistence.xml_ file, or
+- any combination of the previous options.
+
+The managed persistence classes may be:
+
+- contained within the root of the persistence unit,
+- specified by reference—that is, by naming the classes, class archives, or
+  XML mapping files (which in turn reference classes) that are accessible on
+  the application classpath, or
+- specified by some combination of these means.
```

Comment:
_any_  is more appropriate word here; _some_ may trigger thinking that certain combinations are invalid without actually listing them

Reply:
I mean, based on https://dictionary.cambridge.org/grammar/british-grammar/any

_We use any to mean ‘it does not matter which or what’, to describe something which is not limited. We use this meaning of any with all types of nouns and usually in affirmative sentences._

Reply:
I agree, "any" is better here.

Reply:
Fixed.